File: app/models/entities/Employee.go
package entities

// Employee структура сущности работника
type Employee struct {
	ID         int64  `json:"id"`         // идентификатор
	Lastname   string `json:"lastname"`   // фамилия работника
	Firstname  string `json:"firstname"`  // имя работника
	Middlename string `json:"middlename"` // отчество работника
	Position   string `json:"position"`   // должность работника
	//Value string `json:"value"`
}

File: public/js/models/entities/Employee.js
export class Employee {
    constructor(id, lastname, firstname, middlename, position, email) {
        this.id = id
        this.lastname = lastname
        this.firstname = firstname
        this.middlename = middlename
        this.position = position
        this.email = email
        this.value = (lastname + " " + firstname)
    }
}